src/fairchem/core/common/relaxation/ase_utils.py

```diff
@@ -170,6 +171,20 @@ def __init__(
         self.config["checkpoint"] = checkpoint_path
         del config["dataset"]["src"]
 
+        # some models that are published have configs that include tasks
+        # which are not output by the model
+        if only_output is not None:
+            assert isinstance(
```

Comment:
nit, can we do explicit exception raising instead? https://snyk.io/blog/the-dangers-of-assert-in-python/
```python
if not isinstance(output, list):
    raise TypeError("`only_output` must be a list of targets to output")
```